jme3-core/src/main/java/com/jme3/animation/SkeletonControl.java

```diff
@@ -294,8 +294,9 @@ protected void controlRender(RenderManager rm, ViewPort vp) {
 
                 if (hwSkinningSupported) {
                     hwSkinningEnabled = true;
-                    
-                    Logger.getLogger(SkeletonControl.class.getName()).log(Level.INFO, "Hardware skinning engaged for " + spatial);
```

Comment:
Probably moving this statement in the else clause would make the code more readable?